packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -593,7 +593,7 @@ Posts.addView("afRecentDiscussionThreadsList", terms => {
       af: true,
     },
     options: {
-      sort: {lastCommentedAt:-1},
+      sort: {afLastCommentedAt:-1},
```

Comment:
Needs a corresponding index change

Reply:
I updated it with what I think is the correct change, lemme know.